_posts/2017-10-05-denounce-segwit2x.md

```diff
@@ -0,0 +1,75 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+type: posts
+layout: post
+lang: en
+category: blog
+
+title: "Bitcoin.org to denounce Segwit2x"
+permalink: /en/posts/denounce-segwit2x.html
+date: 2017-10-05
+---
+On 2017-10-11 at noon (UTC), Bitcoin.org is planning to publish a banner on every page of the site warning users about the risks of using services that will default to the so-called Segwit2x (S2X) contentious hard fork. S2X companies will be called out by name. To ensure that we only warn users against companies that will actually put user deposits at risk, we urge all companies to publicly clarify their stance before the above date, either by a highly-visible public statement or by commenting on Bitcoin.org issue #[1835](https://github.com/bitcoin-dot-org/bitcoin.org/issues/1835) (or by doing both).
+
+In particular, we need to know that:
+
+1. The company will not under any circumstances list "Segwit2x" as "BTC" and/or "Bitcoin". Note that [Bitcoin is not ruled by miners](https://en.bitcoin.it/wiki/Bitcoin_is_not_ruled_by_miners), and miner actions cannot be used as a justification to redefine Bitcoin.
+2. The company will not by default *convert* user BTC to S2X-coins. Distributing "air-dropped" S2X-coins to users is acceptable, however.
```

Comment:
Suggest removing reference to "bitcoin is not ruled by miners". It adds little to this discussion and the wiki article doesn't really address the current situation fully.

Reply:
I strongly support keeping it.  I think it's a very valuable contribution to this blog post as the NYA cited hash rate support as being necessary (though it wasn't), the segwit signaling part of btc1 required hash rate support (though BIP148 would've activated either way), and S2X supporters frequently point to the current "NYA" signaling of 90%+ of hash rate as heralding their eventual success (though it doesn't).

Reply:
BIP148 itself was controversial and any unilateral uasf like that in the future will also be controversial. Suggesting that users stay on a chain with minimal hashpower can put their coins at risk. The article does not really address the situation of when part of the community wants to change consensus and another part does not except to assume that the part that does want to change the rules is an attack. This is not always the case. 

Linking bitcoin.org to wiki's that are subject to change and editing in general is a bad idea in general.

Reply:
1. I agree BIP148 was controversial (I [did not support it](https://twitter.com/hrdng/status/869360246501847042) and I wrote the Bitcoin.org alert about it/BIP91).
2. The Wiki article does not address protocol upgrades because the Wiki article is not about protocol upgrades.  However, this Bitcoin.org post does link to a [previous Bitcoin.org post](https://bitcoin.org/en/posts/hard-fork-policy) describing what we think are inappropriate hard forks (any hard fork that is contentious).
3. It seems to me that for the past several years, the Wiki seems has been well maintained.  I added the article to my wiki watchlist which I have setup to email me (via my RSS reader) on any changes, so I can help police the article.  If it gets excessively vandalized, we can link to a specific revision.

Reply:
Suggest you just link to the contentious hard fork post then. Although of course it does not define what 'contentious' is or what would not be contentious so it too is lacking but I think more on point to the current situation.

It is rather odd to link to a post like "bitcoin is not ruled by miners" in response to activity by miners. If it is not activity by miners but the corporate entities supporting NYA, then the post completely misses the mark.

I assume the intention is as a reminder to corporations and users that while they are free to run whatever software they want, they should not choose that software based on hash power alone. However, this blog post we are discussing (this pr) is specifically about services and corporations. Services and corporations are not miners. They choose what software to run as suits them. We cannot get inside their heads and say " oh you are doing it just because of hash rate". In fact we as a community are well aware that many services and corps support NYA for other reasons.

Reply:
> Suggest you just link to the contentious hard fork post then.

It is already linked later in the post.